AbletonMCP_Remote_Script/commands/arrangement.py

```
@@ -0,0 +1,490 @@
+        # Schedule the cue point creation using two-step process
+        # Store pending operation for the callback
+        context._pending_cue_create = {"time": target_time, "name": name}
+
+        def step1_set_position():
+            """Step 1: Set position on Ableton's main thread."""
+            try:
+                song.current_song_time = target_time
+                context.log(f"CUE CREATE STEP1: Position set to {target_time}")
+                # Schedule toggle for next frame
+                context.schedule_message(1, step2_toggle)
+            except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)